torch/onnx/utils.py

```diff
@@ -87,7 +87,7 @@ def _split_tensor_list_constants(g, block):
 
 def _optimize_graph(graph, operator_export_type, _disable_torch_constant_prop=False):
     # Inline everyting (recursively)
```

Comment:
Do you want to update the comment as well?

Reply:
Oh, right, thanks (it will become recursive again in #26489, but I think we can just omit that word from the comment).